Configuration/DataProcessing/python/Repack.py

```diff
@@ -55,12 +67,15 @@ def repackProcess(**args):
 
         outputModule = cms.OutputModule(
             "PoolOutputModule",
-            compressionAlgorithm=copy.copy(RAWEventContent.compressionAlgorithm),
-            compressionLevel=copy.copy(RAWEventContent.compressionLevel),
+            compressionAlgorithm=copy.copy(eventContent.compressionAlgorithm),
+            compressionLevel=copy.copy(eventContent.compressionLevel),
             fileName = cms.untracked.string("%s.root" % moduleLabel)
             )
 
-        outputModule.dataset = cms.untracked.PSet(dataTier = cms.untracked.string("RAW"))
+        if dataTier != defaultDataTier:
+            outputModule.outputCommands = copy.copy(eventContent.outputCommands)
```

Comment:
@drkovalskyi - my suggestion is to remove this line and the previous one, which removes the concern I've raised.

Reply:
I agree. Let's not enforce event content in repacking. It introduces more problems then it solves. We may still need the new EventContent for central production, so I would still keep that part, but what concerns repacking itself, I agree with David's proposal.

Reply:
they are needed for MC production and/or derived data sets, but those should be correct if defined... ( i guess the one for hlt scouting is already present somewhere as its in miniaod already iiuc.)

Reply:
For MC uses cases we will have time to fine tune and iterate.

Reply:
please open an issue so that you don't forget to clean things up

Reply:
I've removed these two lines as you suggested.

Reply:
> please open an issue so that you don't forget to clean things up

@drkovalskyi

Please let us know once you open the issue.
Thanks.

Reply:
@haozturk could you please follow up on this request

Reply:
Here is the issue: https://github.com/cms-sw/cmssw/issues/44409 Please correct me if there is any missing or incorrect statements there.